powerhome/playbook
# [PLAY-219] Typescript Conversion: Date Picker

#### Screens

![Screen Shot 2022-09-14 at 2 14 39 PM](https://user-images.githubusercontent.com/8194056/190219701-f6830f0c-225d-4741-b9de-e6ced0e3a7a2.png)

#### Breaking Changes

No breaking changes, typescript conversion of the Date Picker kit

#### Runway Ticket URL

[[PLAY-219]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-219)

#### How to test this

- Check console for any errors
- Try reproducing the kit with different props 
- Run `./test.sh` 

#### Checklist:
